src/app/services/push.service.ts

```diff
@@ -1,15 +1,12 @@
 import { PushRegistration } from '@aerogear/push';
 import { Injectable } from '@angular/core';
-import { Push, PushObject } from '@ionic-native/push/ngx';
 import { ConfigurationService } from '@aerogear/core';
-const config = require('../../mobile-services.json');
 import { Events } from '@ionic/angular';
 import { Storage } from '@ionic/storage';
 
-const PUSH_ALIAS = 'cordova';
+const config = require('../../mobile-services.json');
```

Comment:
We use OpenShift service for this. Should not work with config directly

Reply:
Hrm, this code was already there the IDE just moved it down.